common/tools/dev-tool/tsconfig.json

```diff
@@ -1,12 +1,12 @@
 {
   "compilerOptions": {
-    "target": "ES2019",
+    "target": "ES2021",
```

Comment:
I am curious why is that needed?

Reply:
likely for the string `replaceAll` method